board/safety/safety_volkswagen.h

```diff
@@ -0,0 +1,162 @@
+const int VW_MAX_STEER = 300;               // 3.0 nm
+const int VW_MAX_RT_DELTA = 190;            // max delta torque allowed for real time checks
```

Comment:
can we do 188 precisely? :) otherwise we are going to be confused later on where it came from

Reply:
How about 188 and some free commentary from the peanut gallery?